RUtils/TRY_Harmonise_Utils.r

```diff
@@ -27,624 +27,692 @@
 #                 insensitive).
 #---~---
 TRY_LonLatToGeoInfo <<- function(lon,lat,geo_adm1_path,simplified=TRUE){
```

Comment:
There are quite a bit of differences, but they are mostly just spaces and indentation. I will accept it but I will likely revert to the original indentation after merging it, just to keep it consistent throughout the file.